api/sirsidynix_authentication_provider.py

```diff
@@ -86,15 +72,6 @@ class Keys:
             ),
             "required": True,
         },
-        {
-            "key": Keys.LIBRARY_PREFIX,
-            "label": _("Library Prefix"),
-            "description": _(
-                "This is used to match a member of the Library to their member branch location."
-                " Should be a prefix letter, eg. 'p' or 'co' etc..."
-            ),
-            "required": True,
-        },
```

Comment:
The `LIBRARY_PREFIX` setting is removed, and replaced with the `LIBRARY_IDENTIFIER_RESTRICTION` setting on the base class.